codecov.yml

```diff
@@ -2,12 +2,7 @@ coverage:
   precision: 2
   round: nearest
   range: "60...100"
-  ignore:
-  - LHToolbarDemoCocoaPodsTest/*
-  - LHToolbarTests/*
-  - LHToolbarDemo/*
-  - Pods/*
         
```

Comment:
These will be ignored because of the `-J` below. Codecov will only process that project and skip the ones above.